diff --git a/node_modules/jsonresume-theme-kendall/print.css b/node_modules/jsonresume-theme-kendall/print.css
index a13418f..66555a5 100644
--- a/node_modules/jsonresume-theme-kendall/print.css
+++ b/node_modules/jsonresume-theme-kendall/print.css
@@ -8,7 +8,7 @@ a[href]:after {
 }
 
 #photo{
-  display: none;
+  /* display: none; */
 }
 
 .box {
@@ -16,7 +16,7 @@ a[href]:after {
 }
 
 blockquote,
-#education,
+#education > li,
 #awards,
 .contact-item,
 .publication,
@@ -68,7 +68,7 @@ blockquote,
 }
 
 .badge {
-  margin: 0;
+  /* margin: 0; */
 }
 
 .list-group-item{
@@ -81,11 +81,16 @@ blockquote,
   position: absolute;
   top: 8px;
   right: 0;
-  left: -1px;
+  left: 1mm;
   height: 0;
   width: 0;
   border: solid transparent;
-  border-right-color: #999;
+  border-left-color: #999;
   border-width: 4px;
   pointer-events: none;
- }
+}
+ 
+@page {
+  size: A4;
+  margin: 0.5cm;
+}
diff --git a/node_modules/jsonresume-theme-kendall/resume.template b/node_modules/jsonresume-theme-kendall/resume.template
index e287a61..80be67c 100644
--- a/node_modules/jsonresume-theme-kendall/resume.template
+++ b/node_modules/jsonresume-theme-kendall/resume.template
@@ -16,6 +16,15 @@
     <style type="text/css" media="print">
     {{{printcss}}}
     </style>
+
+    <!-- embed metas - https://ogp.me/ -->
+    <meta property="og:title" content="Alfred Renn" />
+    <meta property="og:type" content="website" />
+    <meta property="og:site" content="Alfred Renn's CV" />
+    <meta property="og:url" content="https://cv.alifeee.co.uk" />
+    <meta property="og:image" content="https://cv.alifeee.co.uk/profile-picture.png" />
+    <meta property="og:description" content="Alfred Renn's CV on the web" />
+    <meta property="og:locale" content="en_GB" />
   </head>
   <body>
     <div class="container">
@@ -30,6 +39,11 @@
             {{/photo}}
             <div id="text-header" {{^photo}}style="margin-top: 90px;"{{/photo}}>
               <h1>{{basics.name}}<br>{{#basics.label}}<span>{{basics.label}}</span>{{/basics.label}}</h1>
+              <span class="visible-print" href="{{website}}" target="_blank">
+                <i class="fas fa-globe ico" ></i>
+                View online!
+                <a href="https://cv.alifeee.co.uk">cv.alifeee.co.uk</a>
+              </span>
             </div>
           </div>
         </div>
