diff --git a/node_modules/jsonresume-theme-kendall/print.css b/node_modules/jsonresume-theme-kendall/print.css
index a13418f..144a3a8 100644
--- a/node_modules/jsonresume-theme-kendall/print.css
+++ b/node_modules/jsonresume-theme-kendall/print.css
@@ -7,16 +7,12 @@ a[href]:after {
   content: none !important;
 }
 
-#photo{
-  display: none;
-}
-
 .box {
   margin-bottom: -10px;
 }
 
 blockquote,
-#education,
+#education > li,
 #awards,
 .contact-item,
 .publication,
@@ -81,11 +77,16 @@ blockquote,
   position: absolute;
   top: 8px;
   right: 0;
-  left: -1px;
+  left: 1mm;
   height: 0;
   width: 0;
   border: solid transparent;
-  border-right-color: #999;
+  border-left-color: #999;
   border-width: 4px;
   pointer-events: none;
- }
+}
+ 
+@page {
+  size: A4;
+  margin: 0.5cm;
+}
diff --git a/node_modules/jsonresume-theme-kendall/resume.template b/node_modules/jsonresume-theme-kendall/resume.template
index e287a61..3ea39c1 100644
--- a/node_modules/jsonresume-theme-kendall/resume.template
+++ b/node_modules/jsonresume-theme-kendall/resume.template
@@ -30,6 +30,11 @@
             {{/photo}}
             <div id="text-header" {{^photo}}style="margin-top: 90px;"{{/photo}}>
               <h1>{{basics.name}}<br>{{#basics.label}}<span>{{basics.label}}</span>{{/basics.label}}</h1>
+              <span class="visible-print" href="{{website}}" target="_blank">
+                <i class="fas fa-globe ico" ></i>
+                View online!
+                <a href="https://cv.alifeee.co.uk">cv.alifeee.co.uk</a>
+              </span>
             </div>
           </div>
         </div>
